core/kernel/asan.c

```diff
@@ -45,6 +46,12 @@ static size_t asan_va_size;
 static bool asan_active;
 static asan_panic_cb_t asan_panic_cb = asan_panic;
 
+static inline bool addr_crosses_scale_boundary(uintptr_t addr, size_t size)
```

Comment:
Please drop the `inline`.
I'd prefer to use `vaddr_t` type when manipulating virtual addresses represented as integers, unless there's a good reason to use another type.

Reply:
> Please drop the inline.

Ok, please read my [comment above](https://github.com/OP-TEE/optee_os/pull/7431#discussion_r2144433782) if the inlines are not needed I will remove them.

> I'd prefer to use vaddr_t type when manipulating virtual addresses represented as integers, unless there's a good reason to use another type.

Done.

Reply:
This `inline` means nothing since it's a static function in the same .c file.